sky/packages/sky/lib/rendering/box.dart

```diff
@@ -1746,15 +1749,22 @@ class RenderView extends RenderObject with RenderObjectWithChildMixin<RenderBox>
   void paintFrame() {
     sky.tracing.begin('RenderView.paintFrame');
     try {
-      double devicePixelRatio = sky.view.devicePixelRatio;
-      sky.PictureRecorder recorder = new sky.PictureRecorder();
-      Rect cullRect = Point.origin & (size * devicePixelRatio);
-      PaintingCanvas canvas = new PaintingCanvas(recorder, cullRect);
-      PaintingContext context = new PaintingContext(canvas);
-      canvas.drawColor(const Color(0xFF000000), sky.TransferMode.src);
-      canvas.scale(devicePixelRatio, devicePixelRatio);
+      final double devicePixelRatio = sky.view.devicePixelRatio;
+
+      // TODO(abarth): Really |_rootLayer| should be a TransformLayer that
+      // applies the devicePixelRatio.
+      Rect scaledBounds = Point.origin & (size * devicePixelRatio);
```

Comment:
size is a |Size|.  Given a Point and a Size, the & operator makes them into a Rect.